.github/workflows/release.yml

```diff
@@ -144,18 +146,18 @@ jobs:
       - name: Checkout
         if: github.event_name != 'pull_request_target'
         uses: actions/checkout@v2
-      - name: Cache Rust dependencies
-        uses: actions/cache@v2
-        id: cache
-        with:
-          path: |
-            ~/.cargo/registry
-            ~/.cargo/git
-            target
-            node/standalone/target
-          key: ${{ runner.OS }}-build-${{ hashFiles('**/Cargo.lock') }}
-          restore-keys: |
-            ${{ runner.OS }}-build
+      # - name: Cache Rust dependencies
+      #   uses: actions/cache@v2
+      #   id: cache
+      #   with:
```

Comment:
We should consider using [sccache](https://github.com/mozilla/sccache) here. Parity has had a lot of success with it and I know @notlesh explored it briefly.

Reply:
I'll give it a try. I wanted to use a simple directory copy as it is always running on the same machine but I can look at this one.

Something to consider with github (that I discovered yesterday) is that the limit of the cache is 5GB, and that is the reason why it is so ineffective on the github workflow currently.

Reply:
I've integrated sccache without playing too much with the settings, result is good, it reduces from 31 to 18 minutes best case.